library/public/utils.h

```diff
@@ -68,6 +68,20 @@ class F3D_EXPORT utils
   [[nodiscard]] static std::filesystem::path collapsePath(
     const std::filesystem::path& path, const std::filesystem::path& baseDirectory = {});
 
+  /**
+   * Converts a glob expression to a regular expression
+   *
+   * It handles the following glob features:
+   * - '*': Matches any number of characters (except path separators)
+   * - '?': Matches exactly one character (except path separators)
+   * - '[...]': Character classes
+   * - '[!...] or '[^...]': Negated character classes
+   * - '**': Matches any number of characters including path separators if enabled
+   * - '{a,b,c}': Alternation (matches any of the comma-separated patterns)
```

Comment:
backticks for IDEs that support Markdown
![image](https://github.com/user-attachments/assets/d32af21c-c0cd-4724-818e-c78439a93c71)
 
```suggestion
   * - `*`: Matches any number of characters (except path separators)
   * - `?`: Matches exactly one character (except path separators)
   * - `[...]`: Character classes
   * - `[!...]` or `[^...]`: Negated character classes
   * - `**`: Matches any number of characters including path separators if enabled
   * - `{a,b,c}`: Alternation (matches any of the comma-separated patterns)
```

Reply:
Added backticks